scripts/java-api-check.sh

```diff
@@ -50,10 +44,7 @@ PREVIOUS_RELEASE=${TMPDIR}/previous/sdk-release.aar
 CURRENT_RELEASE_DIR=$(dirname "${CURRENT_RELEASE}")
 PREVIOUS_RELEASE_DIR=$(dirname "${PREVIOUS_RELEASE}")
 
-gh auth login --with-token < ./gh_token.txt
-LAST_VERSION_TAG=$(gh release list -L 1) #android-v10.3.0
-LAST_VERSION_TAG_ARRAY=($LAST_VERSION_TAG)
-LAST_VERSION=${LAST_VERSION_TAG_ARRAY[0]:9}
+LAST_VERSION=$(grep -o '[0-9]*\.[0-9]*\.[0-9]*' ./gradle.properties) #10.3.0
```

Comment:
will be nice to check only `VERSION_NAME=PATTERN`. because we can store smth else in this file.

Reply:
Updated.